### PR TITLE
BACKLOG-22357: Introduce new tooling to investigate import-package and export-package in the system

### PR DESCRIPTION
**Pull request by jkevan**

### CODE REVIEW

**Review by jayblanc — APPROVED** [review]
Maybe another ticket for a JUnit Test using mocked FrameworkService will allow to test the OSGIPackageHEaderChecker ?

File: src/main/java/org/jahia/modules/tools/gql/admin/ToolsGraphQL.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+@GraphQLName("AdminTools")
+@GraphQLDescription("Jahia admin tools")
+public class ToolsGraphQL {
+    @GraphQLField
+    @GraphQLDescription("Will return all the import packages matching the given parameters.")
+    public FindImportPackage findMatchingImportPackages(
+            @GraphQLName("RegExp") @GraphQLDescription("will return only import-package matching the RegExp") String regExp,
+            @GraphQLName("version") @GraphQLDescription("will return only import-package matching the given version") String version,
+            @GraphQLName("versionMissing") @GraphQLDescription("will return only import-package matching the given version") @GraphQLDefaultValue(GqlUtils.SupplierFalse.class) boolean versionMissing
```
Comment (sergehuber) [inline src/main/java/org/jahia/modules/tools/gql/admin/ToolsGraphQL.java:33]: I think this description is not correct ?

File: src/main/java/org/jahia/modules/tools/gql/admin/osgi/BundleResultEntry.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * limitations under the License.
+ */
+package org.jahia.modules.tools.gql.admin.osgi;
+
+import graphql.annotations.annotationTypes.GraphQLDescription;
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import org.osgi.framework.Bundle;
+
+public abstract class BundleResultEntry {
```
Comment (sergehuber) [inline src/main/java/org/jahia/modules/tools/gql/admin/osgi/BundleResultEntry.java:23]: Could you please add a short class description ?

**Review by sergehuber — CHANGES_REQUESTED** [review]
Hello Kevan, thanks for the PR, nice work !

I just found some minor issues.

**Review by sergehuber — APPROVED** [review]

**Pull request merged**